src/python/bot/fuzzers/ml/gradientfuzz/model_utils.py

```diff
@@ -95,6 +96,7 @@ def update_state(self, *args, **kwargs):
       Computes (true positives) / (true positives + errors).
       """
     y_true, y_pred = args
+    y_true = tf.cast(y_true, tf.bool)
     y_pred = tf.cast(tf.round(y_pred), tf.bool)
```

Comment:
Is this one supposed to casted to bool ?

Reply:
Yeah, since it's used below in logical_and.

Reply:
I should also point out I don't think the cast above is actually doing anything (and it may be cleaner not to check) but I don't mind having the explicit cast while we're getting stuff working initially.